source/javascripts/dependencies.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env */
+
+// Copied without change from the govuk-publishing-components gem
+// to allow use of a custom modules.js with jQuery support
+
+// This adds in javascript that initialises components and dependencies
+// that are provided by Slimmer in public frontend applications.
+// = require ./modules.js
+
+document.addEventListener("DOMContentLoaded", function () {
+  window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {};
+  window.GOVUK.analyticsVars = window.GOVUK.analyticsVars || {};
+
+  // if statements ensure these functions don't execute during testing
+  if (typeof window.GOVUK.loadAnalytics !== "undefined") {
+    window.GOVUK.loadAnalytics.loadExtraDomains();
+    if (typeof window.GOVUK.analyticsGa4.vars === "undefined") {
+      window.GOVUK.loadAnalytics.loadGa4();
+    }
+    if (typeof window.GOVUK.analyticsVars.gaProperty === "undefined") {
+      window.GOVUK.loadAnalytics.loadUa();
+    }
+  }
+
+  window.GOVUK.modules.start();
```

Comment:
I don't think you need this line in here - the tech docs gem has a `document.ready` that calls this anyway. From running your code locally it looks like `start` gets called three times, removing this takes it to 2 (which is still too many, but better).

Reply:
https://github.com/alphagov/tech-docs-gem/blob/main/lib/assets/javascripts/_start-modules.js#L10C1-L12C3

Reply:
Good spot!, thanks. I've removed this line as recommended. Doing this has also fixed the console error message above 😄